README.md

```diff
@@ -155,6 +155,7 @@ source ~/.zshenv
 | Add/remove thumbs-up reaction to the current message  | <kbd>+</kbd>                                  |
```

Comment:
@preetmishra It maybe simpler here to split the commit into 
a) README/keys - which adds the key (without linking it) (`git log --grep='README/keys'`)
b) EditHistory View  - and test for it.
c) Finally, link the keypress function - and tests for it.

Splitting the commits in such a fashion makes reviewing easier and commit much more coherent.